src/web/handlers/FlingGestureHandler.ts

```diff
@@ -132,6 +183,14 @@ export default class FlingGestureHandler extends GestureHandler {
     super.onPointerMove(event);
   }
 
+  protected onPointerMove(event: AdaptedEvent): void {
+    this.pointerMove(event);
+  }
+
+  protected onPointerOutOfBounds(event: AdaptedEvent): void {
+    this.pointerMove(event);
```

Comment:
In `pointerMove` you call `super.onPointerMove()`, but I believe in that case it should be `super.onPointerOutOfBounds()`. I'd remove calling `super` methods in `pointerMove` and call them inside overriden functions.